pom.xml

```diff
@@ -3,7 +3,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.folio</groupId>
   <artifactId>mod-kb-ebsco-java</artifactId>
-  <version>1.0.0-SNAPSHOT</version>
+  <version>0.1.0-SNAPSHOT</version>
   <packaging>jar</packaging>
 
   <licenses>
```

Comment:
While this pom.xml is pretty concrete, there are few interesting tags we can grab from https://github.com/folio-org/mod-codex-ekb/blob/master/pom.xml like name, url, description, organization, issueManagement etc. Also, noticed some differences in <scm> with extensions, <licenses> and so on...Please compare and use your judgement to adjust this accordingly. Thanks!

Reply:
we have used https://github.com/folio-org/mod-rmb-template which creates pom file as well as other boilerplate code. I have added the following tags like ciManagement, issueManagement but the other dependencies seem module-specific and we can add them when it is needed.

Reply:
Sounds good!